src/page-modules/assistant/layout.tsx

```diff
@@ -114,22 +115,14 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
     750,
   );
 
-  const { urls, orgId } = getOrgData();
+  const { orgId } = getOrgData();
   const { isDarkMode } = useTheme();
 
   if (tripQuery.transportModeFilter === null)
     onTransportFilterChanged(setTransportModeFilters(transportModeFilter));
 
   return (
-    <div>
-      <div className={style.homeLink__container}>
-        <ButtonLink
-          mode="transparent"
-          href={urls.homePageUrl.href}
-          title={t(PageText.Assistant.homeLink(urls.homePageUrl.name))}
-          icon={{ left: <MonoIcon icon="navigation/ArrowLeft" /> }}
-        />{' '}
-      </div>
+    <SearchContainerLayout>
```

Comment:
Thought I wrote a comment here, but couldn't find it again, so probably didn't add it 😅. But I was thinking about the naming of this component, and I don't really find it intuitive. I wonder if maybe naming it by intention of use is better than what it is supposed to wrap? So what does this actually do "Search Container Layout" says nothing other than "this wrapped the search inputs", but in reality it shows the global messages.